susemanager/susemanager.changes

```diff
@@ -1,3 +1,4 @@
+- adapt script for migration from 3.2 to 4.0
```

Comment:
I am late for this, but LGTM.

Only thing: when you are adding the first change after a release, remember to ad an empty line between the last header and the new entry :-)